pages/docs/couple-your-code/couple-your-code-global-data.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Exchanging Global Data (data not associated to a mesh)
+permalink: couple-your-code-global-data.html
+keywords: api, adapter, data, global
+summary: "You can define and exchange global data (data not associated to a mesh) with preCICE by using specific optional API functions."
+---
+
+{% warning %}
+These API functions are a work in progress, experimental, and are not yet released. The API might change during the ongoing development process. Use with care.
+{% endwarning %}
+
+{% note %}
+This feature is only available for two-participant coupling and does not yet support [Multi coupling](https://precice.org/configuration-coupling-multi.html). Further, it does not yet allow [Convergence Measures](https://precice.org/configuration-coupling.html#implicit-coupling-schemes) and [Acceleration](https://precice.org/configuration-acceleration) to be defined for global data. An extension to these features is planned.
+{% end note %}
+
+preCICE allows participants to exchange data that is not associated with any mesh. Examples of such data are global background pressure (for e.g. in fluid-acoustic coupling) and angles between coordinate systems (for e.g. in CAMRAD-fluid coupling).
```

Comment:
Can we explain this without pointing to CAMRAD?

Reply:
sure, I just picked it up from the description in https://github.com/precice/precice/issues/202.

How about: "for e.g. in fluid-structure coupling for rotor blades"

Reply:
good enough :+1:

language-wise, I would use 

> (e.g. fluid-structure coupling for rotor blades)

so, no "for" or "in"